src/routes/api.ts

```diff
@@ -1,134 +1,75 @@
 import * as express from "express";
-import pgPromise from "pg-promise";
+import pgPromise, { ParameterizedQuery } from "pg-promise";
 
 export const register = ( app: express.Application ) => {
-	const oidc = app.locals.oidc;
-	const port = parseInt( process.env.PGPORT || "5432", 10 );
-	const config = {
-		database: process.env.PGDATABASE || "postgres",
-		host: process.env.PGHOST || "localhost",
-		port,
-		user: process.env.PGUSER || "postgres"
-	};
+    const oidc = app.locals.oidc;
```

Comment:
It looks like everything is indented four spaces. In my experience, TypeScript and JavaScript is usually two spaces.